eshome/plugins/readonlyrest/plugin-security.policy

```diff
@@ -9,4 +9,6 @@ grant {
   permission java.io.FilePermission "/proc/sys/net/core/somaxconn","read";
   permission java.lang.RuntimePermission "accessClassInPackage.sun.misc";
   permission java.lang.RuntimePermission "setContextClassLoader";
+  permission java.lang.RuntimePermission "createSecurityManager";
```

Comment:
why this privilege is needed?

Reply:
It turned out that these two permissions are needed when I removed using `sec-all.policy` which has `permission java.security.AllPermission;`. All of permissions from `plugin-security.policy` are not needed when `sec-all.policy` is used